resolvePlugins.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+mvn clean test
+groovy steps.groovy
+
+
+[ -d jenkins_home ] && rm -rf jenkins_home
+ cp -r jenkins_home_init jenkins_home
+
+CALLS="`pwd`/jenkins_home/piper/calls.json"
+
+mkdir -p `dirname ${CALLS}`
+
+mv target/performedCalls.json ${CALLS}
+
+[ -f ${CALLS} ] || { echo "File \"${CALLS}\" does not exist." ; exit 1; }
+
+cID=$(docker run -d -v `pwd`/jenkins_home:/var/jenkins_home --env calls=/var/jenkins_home/piper/calls.json --env result=/var/jenkins_home/piper/result.json  ppiper/jenkins-master);
```

Comment:
You could here work with ppiper/jenkinsfile-runner as it is derived from ppiper/jenkins-master but doesn't spin up a complete Jenkins during runtime.

Reply:
Done. Switched to jenkinsfile-runner. It is much faster now.